src/include/optimizer/operators.h

```diff
@@ -141,17 +142,6 @@ class LogicalGroupBy : public OperatorNode<LogicalGroupBy> {
 };
```

Comment:
I know this is not written this time, but do we still want to have the group by columns and having expression stored in the LogicalGroupBy operator instead of store them as physical properties? Have we discussed that? If so, I seem forget the reason for this...